src/googmodule.ts

```diff
@@ -0,0 +1,388 @@
+}
+
+/** Returns true if expr is "module.exports = ...;". */
+function isExportsAssignment(expr: ts.ExpressionStatement): boolean {
+  if (!ts.isBinaryExpression(expr.expression)) return false;
+  if (expr.expression.operatorToken.kind !== ts.SyntaxKind.EqualsToken) return false;
+  return ts.isIdentifier(expr.expression.left) && expr.expression.left.text === 'exports';
+}
+
+/** isUseStrict returns true if node is a "use strict"; statement. */
+function isUseStrict(node: ts.Node): boolean {
+  if (node.kind !== ts.SyntaxKind.ExpressionStatement) return false;
+  const exprStmt = node as ts.ExpressionStatement;
+  const expr = exprStmt.expression;
+  if (expr.kind !== ts.SyntaxKind.StringLiteral) return false;
+  const literal = expr as ts.StringLiteral;
+  return literal.text === 'use strict';
+}
+
+/**
+ * TypeScript inserts the following code to mark ES moduels in CommonJS:
+ *   Object.defineProperty(exports, "__esModule", { value: true });
+ * This matches that code snippet.
+ */
+function isEsModuleProperty(stmt: ts.ExpressionStatement): boolean {
+  // We're matching the explicit source text generated by the TS compiler.
+  // Object.defineProperty(exports, "__esModule", { value: true });
+  const expr = stmt.expression;
+  if (!ts.isCallExpression(expr)) return false;
+  if (!isPropertyAccess(expr.expression, 'Object', 'defineProperty')) return false;
+  // return true;
+  if (expr.arguments.length !== 3) return false;
+  const [exp, esM, val] = expr.arguments;
+  if (!ts.isIdentifier(exp) || exp.escapedText !== 'exports') return false;
+  if (!ts.isStringLiteral(esM) || esM.text !== '__esModule') return false;
+  if (!ts.isObjectLiteralExpression(val) || val.properties.length !== 1) return false;
+  const prop = val.properties[0];
+  if (!ts.isPropertyAssignment(prop)) return false;
+  const ident = prop.name;
+  if (!ident || !ts.isIdentifier(ident) || ident.text !== 'value') return false;
+  return prop.initializer.kind === ts.SyntaxKind.TrueKeyword;
+}
+
+/**
+ * Returns the string argument if call is of the form
+ *   require('foo')
+ */
+function extractRequire(call: ts.CallExpression): string|null {
+  // Verify that the call is a call to require(...).
+  if (call.expression.kind !== ts.SyntaxKind.Identifier) return null;
+  const ident = call.expression as ts.Identifier;
+  if (ident.escapedText !== 'require') return null;
+
+  // Verify the call takes a single string argument and grab it.
+  if (call.arguments.length !== 1) return null;
+  const arg = call.arguments[0];
+  if (arg.kind !== ts.SyntaxKind.StringLiteral) return null;
+  return (arg as ts.StringLiteral).text;
+}
+
+function createGoogCall(methodName: string, literal: ts.StringLiteral): ts.CallExpression {
+  return ts.createCall(
+      ts.createPropertyAccess(ts.createIdentifier('goog'), methodName), undefined, [literal]);
+}
+
+/**
+ * Extracts the namespace part of a goog: import, or returns null if the given
+ * import is not a goog: import.
```

Comment:
Define "import" here, I think you mean the quoted string passed in an import statement and not the import statement itself.